cmd/kube-apiserver/app/server.go

```diff
@@ -227,6 +227,10 @@ func (s *APIServer) verifyClusterIPFlags() {
 	if s.ServiceClusterIPRange.IP == nil {
 		glog.Fatal("No --service-cluster-ip-range specified")
 	}
+	var ones, bits = s.ServiceClusterIPRange.Mask.Size()
+	if bits-ones > 20 {
```

Comment:
should be greater than or equal to since we allow /12.

nit: move the assignment into the if.
```golang
if ones, bits := s.ServiceClusterIPRange.Mask.Size(); bits-ones > 20 {
```